feature-pack/src/main/resources/modules/system/layers/base/org/jboss/weld/spi/main/module.xml

```diff
@@ -34,7 +34,6 @@
 
     <dependencies>
         <module name="javax.annotation.api"/>
```

Comment:
You seem to be missing ``java.desktop`` and ``java.naming`` from the dep list here.

Reply:
JDeps also reports `java.xml.ws.annotation`
I was actually just adding the minimal needed deps and relying on the rest being brought in via different dependencies. But you are right that it should be stated directly here.

Reply:
You can ignore ``java.xml.ws.annotation`` since it's covered by ``javax.annotation.api``.

Reply:
I don't think we have any modules which export ``java.desktop``, unless they're blindly re-exporting ``javax.api`` (which they should not be).